<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inverted Index Search Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Stop word File path : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="leSWPath"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pbSWBrowse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pbAddFile">
      <property name="text">
       <string>Add FIle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pbAddDir">
      <property name="text">
       <string>Add Directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pbBuild">
      <property name="text">
       <string>Build</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QComboBox" name="cbTreeType"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pbSWBuild">
      <property name="text">
       <string>Build Stop Word Tree</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="3" colspan="2">
     <widget class="QListView" name="fileList"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPlainTextEdit" name="cmdOutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLineEdit" name="cmdInput">
      <property name="cursorMoveStyle">
       <enum>Qt::VisualMoveStyle</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
